packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -49,6 +52,26 @@ export interface IconsOptions {
    * Icon displayed in between 2 column headers that allows to resize the column header.
```

Comment:
```suggestion
   * Icon displayed in between two column headers that allows to resize the column header.
```
(Numbers up to ten are always spelled in full)